server/passport-config.js

```diff
@@ -0,0 +1,43 @@
+ import LocalStrategy from "passport-local";
+import bcrypt from "bcrypt";
+import pool from "./db"
+
+const initializePassport = (passport)=>{
+    const authenticateUser =(email, password, done)=>{
+            pool.query("SELECT * FROM users WHERE email = $1", [email])
+            .then(async (result)=>{
+                const user = result.rows[0]
+                if(!user){
+                    return done(null, false, { message: 'Incorrect username or password.' }); 
```

Comment:
Nice error message, this is very good practice for application security to not let the user know which field is incorrect